bug-1671522-pref-assess-change-to-windows-mouse-wheel-scroll-anima-release-82-83.toml

```diff
@@ -0,0 +1,32 @@
+[experiment]
+end_date = '2020-11-17'
+
+segments = ['regular_users_v3', 'is_new_attributed', 'is_not_new_attributed']
+
+## Segments
+
+[segments.is_new_attributed]
+select_expression = '{{agg_any("is_new_attributed")}}'
+data_source = "new_attributed"
+
+[segments.is_not_new_attributed]
+select_expression = '{{agg_any("is_new_attributed")}} = FALSE'
+data_source = "new_attributed"
+
+[segments.data_sources.new_attributed]
+from_expression = """
+(
+    SELECT
+        DATE(submission_timestamp) AS submission_date,
+        client_id,
+        (
+            environment.settings.attribution.medium IS NOT NULL
+            AND DATE_SUB(MIN(environment.profile.creationDate) OVER(PARTITION BY client_id), current_date, INTERVAL DAY) = 0

```

Comment:
`current_date` seems like the wrong value to use here since that's the day the query is run; do you want to use `submission_date` here?

Good news is that this is moot, because attribution and PCD are both available in clients_last_seen, so we should prefer that!